sdk-python/src/apip_sdk_core/__init__.py

```
@@ -0,0 +1,22 @@
+
+from importlib.metadata import PackageNotFoundError, version
+
+from . import policy
+from .policy import *  # noqa: F401,F403
+
+try:
+    __version__: str = version("apip-sdk-core")
+except PackageNotFoundError:
+    __version__ = "0.0.0+unknown"
+
+__all__ = [*policy.__all__, "policy", "__version__"]
```

Ruff (0.15.11)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)